docs/src/pages/guides/guide-react-native/index.md

```diff
@@ -193,7 +193,7 @@ Then:
 {
   "scripts": {
     ...
-    "storybook": "start-storybook -p 6006"
+    "storybook": "adb reverse tcp:7007 tcp:7007 && start-storybook"
```

Comment:
Is this for Android only? Does it cause any problems with iOS?

Reply:
HI, i do not have a ios setup to check with. Will get back to you once i do.